src/FormattingToolbar/toolbarTooltip.js

```diff
@@ -5,28 +5,59 @@
 /**
  * Methods for conversion to strings and further manipulation.
  */
+
 const capitalizeFirst = word => word.toString().charAt(0).toUpperCase();
 
 const sliceWord = word => word.toString().slice(1);
 
 const firstTwoLetters = word => word.toString().slice(0, 2);
 
+
+/**
+ * Function to determine OS and MOD command of user
+ */
+
+function OS () {
+  var userAgent = window.navigator.userAgent,
+      platform = window.navigator.platform,
+      macosPlatforms = ['Macintosh', 'MacIntel', 'MacPPC', 'Mac68K'],
+      windowsPlatforms = ['Win32', 'Win64', 'Windows', 'WinCE'],
+      os = null,
+      MOD = null;
+
+  if (macosPlatforms.indexOf(platform) !== -1) {
+    os = 'Mac OS';
+    MOD = '⌘';
+  }  else if (windowsPlatforms.indexOf(platform) !== -1) {
+    os = 'Windows';
+    MOD = 'Ctrl';
+  } else if (!os && /Linux/.test(platform)) {
+    os = 'Linux';
+    MOD = 'Ctrl';
+  }
+  return MOD; 
+}
+
+export const MOD = OS();
```

Comment:
Some feedback on this:

- It doesn't look like the variable `userAgent` is used.
- `macosPlatforms` and `windowsPlatforms` could potentially be objects with `os` and `MOD` values, so this function could be more functional and return one value rather than assigning a value and returning it.
- I think `let` and `const` should be preferred to `var`
- Could destructure `platform` (`const { platform } = window.navigator;`)

Reply:
```
function OS () {
  const {platform} = window.navigator;

  const macosPlatforms = {
    os: ["Macintosh", "MacIntel", "MacPPC", "Mac68K"],
    MOD: "⌘"
  };

  const windowsPlatforms = {
    os : ["Win32", "Win64", "Windows", "WinCE"],
    MOD: "Ctrl"
  };

  if (macosPlatforms.os.indexOf(platform) !== -1) {
    return macosPlatforms.MOD;
  }  else if (windowsPlatforms.os.indexOf(platform) !== -1) {
    return windowsPlatforms.MOD;
  } else if (!os && /Linux/.test(platform)) {
    return windowsPlatforms.MOD;
  }
}

export const MOD = OS();
```

Reply:
Thanks for the review @irmerk. I hope this is what you meant. Please let me know if there's anything else that needs to be changed.

Reply:
Thanks for the review @irmerk. I'll make these changes and request for review again.